google/cloud/storage/internal/curl_client.cc

```diff
@@ -1250,7 +1250,7 @@ StatusOr<ObjectMetadata> CurlClient::InsertObjectMediaMultipart(
   request.ForEachOption(no_content_type);
 
   // 2. Pick a separator that does not conflict with the request contents.
```

Comment:
Does this comment now overstate the properties of the returned value?

Reply:
PTAL